site/docs/numbers/porting/createBulkPortins.mdx

```diff
@@ -0,0 +1,269 @@
+        <PeerId>521434</PeerId>
+        <ProcessingStatus>DRAFT</ProcessingStatus>
+        <AccountId>9900572</AccountId>
+        <CreatedByUser>jgilmore</CreatedByUser>
+        <LastModifiedBy>jgilmore</LastModifiedBy>
+        <LastModifiedDate>2021-06-21T19:42:34.760Z</LastModifiedDate>
+        <OrderCreateDate>2021-06-21T19:42:34.760Z</OrderCreateDate>
+        <OrderId>ac2c8ab2-7a63-44da-a307-edcabe0b6c81</OrderId>
+    </BulkPortin>
+</BulkPortinResponse>
+```
+
+</TabItem>
+</Tabs>
+
+When a Bulk Port-in is created, it has ```DRAFT``` status, which means it has not been processed yet.
+If you want to update your Bulk Port-in (either in ```DRAFT``` or other further state) follow the [How to update Bulk Port-in](/docs/numbers/guides/porting/updateBulkPortins) guide.
+
+## Provide telephone number(s) to port
+
+After the Bulk Port-in is created, add telephone numbers to the order by sending a <Highlight color="#9a59c5">PUT</Highlight> request to our [API /bulkPortins/{orderId}/tnList](/apis/numbers/#operation/UpdateBulkTnList) endpoint. This can be done through tools like Postman, or in command line.
+
+<Highlight color="#9a59c5">PUT</Highlight> <code>https://dashboard.bandwidth.com/api/accounts/{accountId}/bulkPortins/{orderId}/tnList</code>
+<Tabs
+    groupId="putBulkPortinTnList"
+    defaultValue="payload"
+    values={[
+        { label: 'Request Payload', value: 'payload', },
+        { label: 'cURL', value: 'curl', },
+    ]
+    }>
+<TabItem value="payload">
+
+> Request
+
+```xml
+Note: Make sure an authentication header is added with your BANDWIDTH_USERNAME and BANDWIDTH:PASSWORD
+
+https://dashboard.bandwidth.com/api/accounts/{accountId}/bulkPortins/{orderId}/tnList
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\n
+<TnList>
+    <TN>[telephone number]</TN>
+    <TN>[telephone number]</TN>
+    <!-- SNIP -->
+    <TN>[telephone number]</TN>
+    <TN>[telephone number]</TN>
+</TnList>
+```
+
+> Response
+
+```xml
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<TnListResponse>
+    <OrderId>ac2c8ab2-7a63-44da-a307-edcabe0b6c81</OrderId>
+    <ProcessingStatus>VALIDATE_DRAFT_TNS</ProcessingStatus>
+</TnListResponse>
+```
+
+</TabItem>
+<TabItem value="curl">
+
+```xml
+Note: Remember to add authentication for your application if needed!
+
+curl 'https://dashboard.bandwidth.com/api/accounts/{accountId}/bulkPortins/{orderId}/tnList' \
+-X POST \
+-U '{BANDWIDTH_USERNAME}:{BANDWIDTH:PASSWORD}' \
+-H 'Content-Type: application/xml' \
+-d '<TnList>
+    <TN>[telephone number]</TN>
+    <TN>[telephone number]</TN>
+    <!-- SNIP -->
+    <TN>[telephone number]</TN>
+    <TN>[telephone number]</TN>
+</TnList>'
+```
+
+> Response
+
+```xml
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<TnListResponse>
+    <OrderId>ac2c8ab2-7a63-44da-a307-edcabe0b6c81</OrderId>
+    <ProcessingStatus>VALIDATE_DRAFT_TNS</ProcessingStatus>
+</TnListResponse>
+```
+
+</TabItem>
+</Tabs>
+
+### Polling vs. Webhooks
```

Comment:
Suggest making this header something like "Checking the Decomposition Status of a Bulk Port-in" and describe what that means (i.e. the phone numbers are validated and child port-ins are created).  Then create sections on the two methods of checking that decompositions status either through polling or through a subscription.

Reply:
Addressed. Does it look ok?

![Screenshot 2022-08-30 at 14 11 55](https://user-images.githubusercontent.com/5918477/187422257-f21f44f9-8da4-4e11-8a84-e29e1fcc1018.png)

![Screenshot 2022-08-30 at 14 12 28](https://user-images.githubusercontent.com/5918477/187422333-98940cd9-c691-4fcf-bf74-ba43442f5961.png)

Reply:
Looks good.